web/html/src/manager/login/login.tsx

```diff
@@ -54,6 +54,23 @@ const getGlobalMessages = (validationErrors, schemaUpgradeRequired) => {
     messages = messages.concat({ severity: "error", text: schemaUpgradeError });
   }
 
+  if (diskspaceSeverity == "misconfiguration") {
+    const directoryMissing = t(
+      "Some important directories are missing. Please review your server configuration."
```

Comment:
Not all user of Uyuni/SUSE Manager have ssh access to the system and can do such a review/change.
Maybe we should add a generic advice "Please contact your system administrator to review the server configuration" or on the other messages below:
"Please contact your system administrator to add more disk space".

Reply:
Good point. I reworded the messages.